ipatests/test_integration/test_dns_locations.py

```diff
@@ -61,13 +61,14 @@ def resolve_records_from_server(rname, rtype, nameserver):
     res.lifetime = 30
     logger.info("Query: %s %s, nameserver %s", rname, rtype, nameserver)
     # lets try to query 3x
-    for _i in range(3):
+    for _i in range(10):
         try:
             ans = res.resolve(rname, rtype)
             logger.info("Answer: %s", ans.rrset)
             return ans.rrset
         except (dns.resolver.NXDOMAIN, dns.resolver.Timeout) as e:
             error = e
+            logger.info("Retry[%d]: %s %s, nameserver %s (REASON:%s)" % (_i, rname, rtype, nameserver, error))
```

Comment:
The line is too long, and pylint reports the following issue:
```
************* Module ipatests.test_integration.test_dns_locations
ipatests/test_integration/test_dns_locations.py:71: [W1201(logging-not-lazy), resolve_records_from_server] Use lazy % formatting in logging functions)
```

Reply:
Hi @flo-renaud, I'm wondering that tasks.wait_for_replication can wait until the replicate is updated with nsupdate about a specific entry.The replica has already started, but it's required to wait the dns entry is propagated into replica. Was it possible with tasks.wait_for_replication ?

Reply:
Hi @mmatsuya 
tasks.wait_for_replication checks for the global replication state (all updates have been propagated), it is not able to check if a given entry has been replicated. But it's good enough for the purpose of the test IMO.

Reply:
Thanks, @flo-renaud. I updated the commit.